packages/devtools-source-map/src/index.js

```diff
@@ -11,7 +11,7 @@ const {
 const { workerUtils: { workerTask }} = require("devtools-modules");
 const { setConfig, getValue } = require("devtools-config");
 
-import type { Location } from "devtools-client-adapters/src/types";
+import type { Location } from "debugger-html";
```

Comment:
`devtools-source-map` doesn't depend on the debugger, so it won't have access to this, if I am understanding correctly. Did you learn anything more about how types are shared in the Flow community?